jsapp/js/projects/projectViews/utils.tests.ts

```diff
@@ -71,12 +71,10 @@ describe('projectViewsUtils', () => {
           out: 'settings__description__istartswith:"foo"',
         },
         {
-          in: {
-            fieldName: 'ownerUsername',
-            condition: 'doesNotContain',
-            value: 'foo',
-          },
-          out: 'NOT owner__username__icontains:"foo"',
+          in: {fieldName: 'ownerUsername', condition: 'contains', value: 'foo'},
+          out:
+            '(search_field__owner_username__icontains:"foo" ' +
+            'OR search_field__organization_name__icontains:"foo")',
```

Comment:
in this update, it looks like "owner does not contain" was removed and "owner does contain" was added.

- does the frontend support **exact match** on owner name? if so, should there be a test for this one, too?
- (less important) should we add back the does not contain test?